### ISSUE TITLE
Version 0.8.874 broken on IE11

### ISSUE THREAD

**Issue by fmagaldea** [cognitect/transit-js#57]
Seems this version is not compatible with legacy browser IE11.
It works with any other browser.

Sadly I can't say much as my debug environment is so slow that I can't see the JS error in IE11 debug tools, but I tried with the previous version of your package (reverting manually inside package-lock.json file) and everything works.

In my project this package is used through `transit-immutable-js` and `redux-persist-transform-immutable`.

**Comment by vincentol** [comment]
Adding some more findings since I just ran into this issue.
The error I got was "const must be initialized" coming from a line that looks like this:
```
  a("es6", function() {
    return goog.isEdge_() ? !1 : b('(()=>{"use strict";class X{constructor(){if(new.target!=String)throw 1;this.x=42}}let q=Reflect.construct(X,[],String);if(q.x!=42||!(q instanceof String))throw 1;for(const a of[2,3]){if(a==2)continue;function f(z={a}){let a=0;return z.a}{function f(){return 0;}}return f()==3}})()');
  });
  ```
  For anyone scouring the web for a quick solution, downgrading transit-js works to solve the issue:
  in package.json
```
...
    "redux-persist-immutable": "^4.3.1",
    "transit-js": "0.8.846",
...
```

**Comment by fmagaldea** [comment]
Good to know!
You could fix up to version `0.8.867`, only the next one is broken.
